modules/installation-program-auto-create-subnets.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+
+[id="installation-program-auto-create-subnets_{context}"]
+= Setting the installation program to automatically create subnets
+
+// This feature is related to the edge compute pool only?
+
+You can set the installation program to automatically create public and private subnets for each Local Zone region that you defined in your installation configuration file. This automation also extends to the installation program running nodes in each Local Zone region that contains `machineset` manifest files.
```

Comment:
I see that the following link details differences between private and public subnets. I need to distinguish this?

[Link](https://github.com/openshift/enhancements/pull/1232/files#:~:text=Create%20Machine%20set,in%20Local%20Zones)

Reply:
I don't think so.

(the link points directly to the EP)

Reply:
I think it was resolved in the last review. @dfitzmau please double check.